packages/build/src/plugins/compatibility.ts

```diff
@@ -1,4 +1,3 @@
-import _pEvery from 'p-every'
```

Comment:
Can we also drop this dependency from `package.json` now?

Reply:
you're right! i missed that. i've pushed the change